client/elements/sc-donate-now-page.html

```diff
@@ -440,6 +439,27 @@
                   oneTimeDonation.checked = false;
                   monthly.checked = true;
               });
+              
+              let amountInput = this.shadowRoot.querySelector('.amount-input'); 
+              amountInput.onpaste = (e) => {  // Prevent from text data.
```

Comment:
Comment should be above this line. IMHO `Don't allow pasting text data` sounds more correct.